parser_library/test/workspace/workspace_fade_test.cpp

```diff
@@ -244,11 +241,58 @@ TEST_P(fade_fixture_opencode_macros_opencode, macros_opencode)
 }
 
 namespace {
-class fade_fixture_opencode_macros_opencode_inner : public fade_fixture_opencode_base
+class opencode_deferred_fixture : public fade_fixture_base
 {};
 
 INSTANTIATE_TEST_SUITE_P(fade,
-    fade_fixture_opencode_macros_opencode_inner,
+    opencode_deferred_fixture,
+    ::testing::Values(test_params {
+        {},
+        {
+            fade_message_s::inactive_statement("src1.hlasm", range(position(2, 0), position(2, 72))),
+            fade_message_s::inactive_statement("src1.hlasm", range(position(4, 0), position(4, 72))),
+            fade_message_s::inactive_statement("src1.hlasm", range(position(6, 0), position(6, 72))),
+        },
+    }));
+} // namespace
+
+TEST_P(opencode_deferred_fixture, opencode_deferred)
+{
+    static const std::string src = R"(
+         AIF (L'X EQ 4).SKIP1
+         SAM31
+.SKIP1   ANOP
+
```

Comment:
is this empty line faded?

Reply:
Unfortunately yes. I don't know of any good way at the moment which could prevent this without breaking other stuff...

Reply:
I tested it with AINSERT which looked fine but I'll test it with AREAD as well 👍

Reply:
Ongoing